.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
Should this have `.git` in it too?

Reply:
Hmm, good question. I can check the resultant image, see if it's in there, and remove it if that doesn't break anything.